src/components/Photo/Selfie.js

```diff
@@ -55,8 +55,12 @@ export default class Selfie extends Component<Props, State> {
     }))
   }
 
-  takeSnapshot = () =>
-    this.webcam && screenshot(this.webcam, this.handleSnapshot)
+  takeSnapshot = async() => {
+    const webcam = await this.webcam
```

Comment:
`this.webcam` isn't a promise here, so wont the await will just be resolving to null? On which note, is this also a similar scenario to above?